docs/ops/sort/MatrixNMS_8.md

```diff
@@ -0,0 +1,157 @@
+##  MatrixNms<a name="MatrixNms"></a> {#openvino_docs_ops_sort_MatrixNms_8}
```

Comment:
Let's rename operation to `MatrixNonMaxSuppression` to be consistent with already existing `NonMaxSuppression` operation.